src/App.jsx

```diff
@@ -72,13 +72,13 @@ class App extends Component {
       firstTime: true,
       onboarded: cookies.get('_onboarded'),
       instructionIsOpen: (cookies.get('_onboarded') == "true") ? false : true,
-      age: cookies.get('age'),
+      age: cookies.get('age') >= 18 && cookies.get('age') <= 150 ? cookies.get('age') : '',
```

Comment:
remove this line since we are using some different approach

Reply:
I have removed this as per your request.